pkgs/development/tools/verible/default.nix

```
@@ -0,0 +1,80 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, buildBazelPackage
+, bazel_4
+, flex
+, bison
+, python3
+}:
+
+buildBazelPackage rec {
+  pname = "verible";
+  version = "v0.0-2172-g238b6df6";
+
+  # These environment variables are read in bazel/build-version.py to create
+  # a build string. Otherwise it would attempt to extract it from .git/.
+  GIT_DATE = "2022-08-08";
+  GIT_VERSION = "${version}";
+
+  src = fetchFromGitHub {
+    owner = "chipsalliance";
+    repo = "verible";
+    rev = "${version}";
+    sha256 = "sha256-iOJhdbipuqqBiYGgk95d1c8bEK6Z16l16GuzYCQRc2g=";
+  };
+  patches = [ ./remove-unused-deps.patch ];    # TODO: fix upstream
+
+  bazelFlags = [ "--//bazel:use_local_flex_bison" ];
+
+  fetchAttrs = {
+    # Fixed output derivation hash after bazel fetch
+    sha256 = if stdenv.isDarwin then
+        "sha256-JjibpdSkz3eiO/PCRx0D7h5u3Q8R44InRyqW9mJSMHo="
+       else
+        "sha256-XoLdlEeoDJlyWlnXZADHOKu06zKHgHJfgey8UhOt+LM=";
+  };
+
+  nativeBuildInputs = [
+    flex       # We use local flex and bison as WORKSPACE sources fail
+    bison      # .. to compile with newer glibc
+    python3
+  ];
+
+  preBuild = ''
```

```suggestion
  postPatch = ''
```
Also which files do really need patching?

---

Done.
The WORKSPACE file needs to be patched, I now added a comment above in the patch section to clarify why that is needed.